README.md

```diff
@@ -339,7 +339,7 @@ This repository also provides one such platforms where contributers come over an
 | [Rapid_click_frenzy](https://github.com/kunjgit/GameZone/tree/main/Games/Rapid_click_frenzy) 
 |[Penguins Can't Fly](https://github.com/Will2Jacks/GameZoneForked/tree/Task/Games/Penguins_Can't_Fly)|
 | [Taash_Game](https://github.com/kunjgit/GameZone/tree/main/Games/Taash_Game)|
-
+| [Tetris Game](https://github.com/kunjgit/GameZone/tree/main/Games/tetris_game)|
```

Comment:
place your game entry 10places above it is giving merge conflicts.

Reply:
done